plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Blobstores.js

```diff
@@ -155,6 +155,27 @@ Ext.define('NX.coreui.controller.Blobstores', {
   },
 
   /**
+   * @protected
+   * Enable 'Delete' when user has 'delete' permission, blob store is not in use and it does not have any blob.
+   */
+  bindDeleteButton: function(button) {
+    var me = this;
+    button.mon(
+        NX.Conditions.and(
+            NX.Conditions.isPermitted(me.permission + ':delete'),
+            NX.Conditions.gridHasSelection(me.masters[0], function(model) {
+              return !model.get('inUse') && model.get('blobCount') === 0;
```

Comment:
Do we need a corresponding server-side check, so a stale UI can't delete a blob store if another user has just associated it with a repository?